packages/DataGrid/stories/DataGrid.zebra.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import * as Sbook from "storybook/assets/styles/common.styles";
+import worker from "workerize-loader!./helpers/data.worker"; // eslint-disable-line import/no-webpack-loader-syntax
+import Spinner from "@paprika/spinner";
+import DataGrid from "../src";
+
+export function App() {
+  const [data, setData] = React.useState([]);
+  const [isIdle, setIsIdle] = React.useState(true);
+  const refDataGrid = React.useRef(null);
+
+  React.useEffect(() => {
+    async function loadData() {
+      const w = worker();
+      const data = await w.getDataFromWorker(100, 5);
+      setData(() => data);
+      setIsIdle(() => false);
+    }
+
+    loadData();
+  }, []);
+
+  React.useEffect(() => {
+    if (data.length > 0) {
```

Comment:
just `data.length`?

Reply:
yes, just checking if data has item, is just an story is running in a controlled env. :)